stanford_profile.install

```diff
@@ -69,6 +70,41 @@ function stanford_profile_install() {
   $shortcut->save();
 
   // Allow all users to use search.
-  user_role_grant_permissions(RoleInterface::ANONYMOUS_ID, ['search content']);
-  user_role_grant_permissions(RoleInterface::AUTHENTICATED_ID, ['search content']);
+  user_role_grant_permissions(RoleInterface::ANONYMOUS_ID, [
+    'access content',
+    'search content',
+  ]);
+  user_role_grant_permissions(RoleInterface::AUTHENTICATED_ID, [
+    'access content',
+    'search content',
+  ]);
+
+  if (\Drupal::moduleHandler()->moduleExists('default_content')) {
```

Comment:
I seem some additional error handling, but why was the content import moved to the install rather than listed in the .info.yml?

Reply:
error handling shows messages in the function that is called.. the lines in info.yml dont do anything. In fact by using `config_installer` default content doesnt get created. We have to explicitly call to install them